### PR TITLE
Tomcat server added

### PR DESCRIPTION
**Pull request by KovalAlexey**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -54,4 +73,4 @@
             </plugins>
         </pluginManagement>
     </build>
-</project>
+</project>
```
Comment (tkhambir) [inline pom.xml]: add empty line here

File: src/main/webapp/WEB-INF/web.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    <servlet>
+        <servlet-name>index</servlet-name>
+        <servlet-class>com.internet.shop.controllers.IndexController</servlet-class>
+    </servlet>
+    
+    <servlet-mapping>
+        <servlet-name>index</servlet-name>
+        <url-pattern>/index</url-pattern>
+    </servlet-mapping>
+</web-app>
```
Comment (tkhambir) [inline src/main/webapp/WEB-INF/web.xml]: there is no empty line too

File: src/main/java/com/internet/shop/service/OrderService.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    Order completeOrder(ShoppingCart shoppingCart);
+
+    Order get(Long id);
+
+    List<Order> getUsersOrders(Long userId);
+
+    List<Order> getAll();
+
+    boolean delete(Long id);
+}
```
Comment (tkhambir) [inline src/main/java/com/internet/shop/service/OrderService.java:18]: maybe, will be better to create GenericService

File: src/main/webapp/WEB-INF/web.xml (hunk 2 above)
Comment (KovalAlexey) [reply src/main/webapp/WEB-INF/web.xml]: done

File: pom.xml (hunk 1 above)
Comment (KovalAlexey) [reply pom.xml]: done

**Pull request closed without merging**